langchain4j-mongodb-atlas/src/main/java/dev/langchain4j/store/embedding/mongodb/IndexMapping.java

```diff
@@ -0,0 +1,28 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class IndexMapping {
+
+    private int dimension;
+    private Set<String> metadataFieldNames;
+
+    public static IndexMapping defaultIndexMapping() {
+        Set<String> metadataFieldNames = new HashSet<>(1);
+        metadataFieldNames.add("type");
+        return IndexMapping.builder()
+                .dimension(384)
```

Comment:
I am wondering: should we set this to 1536? Most people are using OpenAI by default AFAIK.

Reply:
good point, already change it to 1536